### DISCUSSION TITLE
k8s deployment

### DISCUSSION THREAD

**Discussion by sufiyanpk7** [getcandy/api-boilerplate#4] [Ideas]
@glennjacobs @alecritson 
If we build a helm chart for getcandy, people will be able to use it quickly, let us try  if isn't  already exists😊

**Comment by alecritson** [comment]
Hi @sufiyanpk7 

We're currently full steam ahead with v2 at this point, which will have a totally different set up. For that reason I'd hate for you or others to spend time on something that would become instantly deprecated.

We're looking at releasing a dev preview of v2 in November so you will be able to get your hands on it then and decide if something like a helm chart would still be useful.

**Reply by sufiyanpk7** [reply]
@alecritson Thanks for having time to write back. Surely we shall see  v2 release  and  will let you know
